src/components/splash/Splash.jsx

```diff
@@ -40,6 +42,20 @@ export const resetLastSplash = async () => {
 const Splash = ({ animation, isLoggedIn }) => {
   const [checked, setChecked] = useState(false)
   const [shouldAnimate, setShouldAnimate] = useState(isLoggedIn !== true || isMobileNative)
+  const { showDeprecationDialog } = useDeprecationDialog()
+
+  useEffect(() => {
```

Comment:
limit the amount of times you display it or some kind of exponential backoff

Reply:
added tho following backoff:
1. shows first 3 times
2. gradually lowers the probability of it showing
3. at some point reset.

Good enough?

Reply:
https://github.com/GoodDollar/GoodDAPP/pull/4296/commits/b53f24c6679940426f925e5b301444bc3d267d35